src/Sdk/WebApi/WebApi/BrokerHttpClient.cs

```diff
@@ -87,6 +88,11 @@ public class BrokerHttpClient : RawHttpClientBase
                 queryParams.Add("architecture", architecture);
             }
 
+            if (disableUpdate != null)
+            {
+                queryParams.Add("disableUpdate", disableUpdate.Value.ToString().ToLower());
```

Comment:
do we need to always provide `disableUpdate=false` as default, or the service will handle the default case when the request doesn't have the query parameter?

Reply:
yeah we assume false when it's not there